doc/sphinx/proofs/writing-proofs/equality.rst

```diff
@@ -976,12 +974,17 @@ which supports additional fine-tuning.
 
 .. cmd:: Opaque {? ! } {+ @reference }
 
-   Marks the specified constants as :term:`opaque` so tactics won't :term:`unfold` them
-   with :term:`delta-reduction`.
+   Marks the specified constants as :term:`opaque` so that most tactics won't
+   :term:`unfold` them with :term:`delta-reduction`.
```

Comment:
```suggestion
   Marks the specified constants as :term:`opaque` so that most reduction tactics won't
   :term:`unfold` them with :term:`delta-reduction`.
```

Reply:
I've tended to refer to "conversion tactics", e.g. [here](https://coq.inria.fr/doc/V8.20.0/refman/proofs/writing-proofs/equality.html#applying-conversion-rules).  "reduction" seems less specific.